src/createTippy.ts

```diff
@@ -215,7 +219,7 @@ export default function createTippy(reference, collectionProps) {
       }),
       clientWidth: 0,
       clientHeight: 0,
-    }
+    } as ReferenceElement
```

Comment:
It's actually not a bug because Popper only uses those three properties (and we don't override the Tippy instance's reference, only the Popper's. But still, I'll merge the previous object in 😄